[FATAL] Non-resolvable parent POM for org.apache.shardingsphere:shardingsphere-sql-parser-opengauss:5.0.0-RC1-SNAPSHOT: Could not find artifact org.apache.shardingsphere:shardingsphere:pom:5.0.0-RC1-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 22, column 13

When I run ./mvnw clean install -Prelease, below is the exception I met with the latest master code:

[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for org.apache.shardingsphere:shardingsphere-sql-parser-opengauss:5.0.0-RC1-SNAPSHOT: Could not find artifact org.apache.shardingsphere:shardingsphere:pom:5.0.0-RC1-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 22, column 13
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.apache.shardingsphere:shardingsphere-sql-parser-opengauss:5.0.0-RC1-SNAPSHOT (/Users/wenwu/workspace/shardingsphere/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for org.apache.shardingsphere:shardingsphere-sql-parser-opengauss:5.0.0-RC1-SNAPSHOT: Could not find artifact org.apache.shardingsphere:shardingsphere:pom:5.0.0-RC1-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 22, column 13 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/pom.xml
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-opengauss/pom.xml
@@ -20,8 +20,8 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>shardingsphere</artifactId>
         <groupId>org.apache.shardingsphere</groupId>
+        <artifactId>shardingsphere-sql-parser-dialect</artifactId>
         <version>5.0.0-RC1-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>

